packages/demo/frontend/src/utils/balanceMatching.ts

```diff
@@ -66,21 +66,28 @@ export function matchAssetBalance({
     )
   }
 
+  const isWeth =
+    selectedAssetSymbol === 'WETH' || selectedAssetSymbol.includes('ETH')
```

Comment:
similar question here as above, are we considering both ETH and WETH to be WETH?

Reply:
Made this only WETH as well.